ibm/service/eventnotification/resource_ibm_en_smtp_user.go

```diff
@@ -45,6 +45,11 @@ func ResourceIBMEnSMTPUser() *schema.Resource {
 				Optional:    true,
 				Description: "SMTP confg Id.",
 			},
+			"username_to_clone": &schema.Schema{
```

Comment:
This doesn't have acceptance tests changes ?

Reply:
Added acceptance code changes